src/Converter/GhostscriptConverter.php

```diff
@@ -44,10 +44,7 @@ public function __construct(GhostscriptConverterCommand $command, Filesystem $fs
     {
         $this->command = $command;
         $this->fs = $fs;
-
-        if ($tmp) {
-            $this->tmp = $tmp;
-        }
+        $this->tmp = empty($tmp)?sys_get_temp_dir():$tmp;
```

Comment:
if you don't care about 5.2.1 support (travis is starting from 5.3) I can also write the shorthand for that